File: src/leetcode/LengthOfLastWord.java
/*
 * Copyright (c) 2023 THGEEK.COM. All rights reserved.
 */

package leetcode;

/**
 * LeetCode 58
 *
 * @author Tao Hong
 * @version 1.0
 * @since 2023/12/17 23:18
 */
public class LengthOfLastWord {
    public int lengthOfLastWord(String s) {
        String[] words = s.split(" ");
        return words[words.length - 1].length();
    }
}

File: src/leetcode/e58/LengthOfLastWord.go
package e58

import "strings"

// LeetCode 58
func lengthOfLastWord(s string) int {
	words := strings.Fields(s)
	wordCnt := len(words)
	if wordCnt == 0 {
		return 0
	}
	return len(words[wordCnt-1])
}
